pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                               set_cmd=sdisable_cmd,
+                               # vals=vals.Bool()
+                               )
+            sholdoff_cmd = 'qutech:input{}:holdoff'.format(ch_pair)
+            self.add_parameter('ch_pair{}_inavg_holdoff'.format(ch_pair),
+                               label=('set holdoff' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=sholdoff_cmd + '?',
+                               set_cmd=sholdoff_cmd + ' {}',
+                               vals=vals.Numbers(0, 127)
+                               )
+            '''
+            Weighted integral + Rotation matrix + TV mode parameters
+            '''
+            swinten_cmd = 'qutech:wint{}:enable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_wint_enable'.format(ch_pair),
+                               label=('enable wighted integral' +
+                                      'chpair {} '.format(ch_pair)),
+                               get_cmd=swinten_cmd + '?',
+                               set_cmd=swinten_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            swintdis_cmd = 'qutech:wint{}:disable'.format(ch_pair)
+            self.add_parameter('ch_pair{}_wint_disable'.format(ch_pair),
+                               label=('Disable wighted integral' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=swinten_cmd + '?',
+                               set_cmd=swintdis_cmd,
+                               # vals=vals.Bool()
+                               )
+            sintlength_cmd = 'qutech:wint{}:intlength'.format(ch_pair)
+            self.add_parameter('ch_pair{}_wint_intlength'.format(ch_pair),
+                               units='#',
+                               label=('the number of sample pairs that' +
+                                      'is used for one integration of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=sintlength_cmd + '?',
+                               set_cmd=sintlength_cmd + ' {}',
+                               vals=vals.Numbers(1, 2048)
+                               )
+            swintstat_cmd = 'qutech:wint{}:status'.format(ch_pair)
+            self.add_parameter('ch_pair{}_wint_status'.format(ch_pair),
+                               units='#',
+                               label=('weighted integral status of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=swintstat_cmd + '?'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat00_cmd = 'qutech:rotmat{}:rotmat00'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat00'.format(ch_pair),
```

Comment:
I would propose 
`ch_pair{}_rotmat_00` to make the parameter name a bit shorter